src/main/resources/avro/referencemethods.avdl

```diff
@@ -113,9 +113,9 @@ as JSON.
 */
 record SearchReferencesRequest {
   /**
-  If present, return only references which belong to this reference set.
+  The ReferenceSet to search.
   */
-  union { null, string } referenceSetId = null;
```

Comment:
Thanks @richarddurbin, that's helpful.

@calbach --- I've reverted the change in question.